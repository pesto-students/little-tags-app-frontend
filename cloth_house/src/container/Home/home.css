.offersheading{
    font-size: medium;
}
.content{
    padding-bottom: 3%;
}