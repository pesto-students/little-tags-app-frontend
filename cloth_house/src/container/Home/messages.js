/*
 * Hubspot Messages
 *
 * This contains all the text for the  Hubspot component.
 */

import { defineMessages } from "react-intl";

export default defineMessages({
  homeHeading: {
    id: "homepage.welcome",
    defaultMessage: "Go to Test"
  }
});
