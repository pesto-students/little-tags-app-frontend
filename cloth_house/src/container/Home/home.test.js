import createComponentWithIntl from '../../utils/helper_snapshot'
import Home from "./index"

it('Home Snapshot', () => {
    const component = createComponentWithIntl(<Home />);
    let tree = component.toJSON()
  
    expect(tree).toMatchSnapshot()
  
  });