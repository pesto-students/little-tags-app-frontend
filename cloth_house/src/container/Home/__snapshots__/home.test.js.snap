// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Snapshot 1`] = `
<section
  className="ant-layout layout"
>
  <header
    className="ant-layout-header header"
  >
    <div
      className="ant-row"
      style={
        Object {
          "marginBottom": undefined,
          "marginLeft": undefined,
          "marginRight": undefined,
          "marginTop": undefined,
        }
      }
    >
      <div
        className="ant-space ant-space-horizontal ant-space-align-center"
        style={Object {}}
      >
        <div
          className="ant-space-item"
          style={
            Object {
              "marginRight": 8,
            }
          }
        >
          <div
            className="logo"
            onClick={[Function]}
          >
            SmileBag
             
            <img
              alt="fallback"
              className="logoImage"
              src="./assets/images/Raster.png"
            />
          </div>
        </div>
        <div
          className="ant-space-item"
          style={
            Object {
              "marginRight": 8,
            }
          }
        >
          <div
            className="searchInput"
          >
            <span
              className="ant-input-affix-wrapper ant-input-affix-wrapper-lg"
              onMouseUp={[Function]}
            >
              <input
                className="ant-input ant-input-lg"
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                placeholder="Search"
                style={null}
                type="text"
                value=""
              />
              <span
                className="ant-input-suffix"
              >
                <span
                  aria-label="search"
                  className="anticon anticon-search"
                  onClick={[Function]}
                  role="img"
                  tabIndex={-1}
                >
                  <svg
                    aria-hidden="true"
                    data-icon="search"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    viewBox="64 64 896 896"
                    width="1em"
                  >
                    <path
                      d="M909.6 854.5L649.9 594.8C690.2 542.7 712 479 712 412c0-80.2-31.3-155.4-87.9-212.1-56.6-56.7-132-87.9-212.1-87.9s-155.5 31.3-212.1 87.9C143.2 256.5 112 331.8 112 412c0 80.1 31.3 155.5 87.9 212.1C256.5 680.8 331.8 712 412 712c67 0 130.6-21.8 182.7-62l259.7 259.6a8.2 8.2 0 0011.6 0l43.6-43.5a8.2 8.2 0 000-11.6zM570.4 570.4C528 612.7 471.8 636 412 636s-116-23.3-158.4-65.6C211.3 528 188 471.8 188 412s23.3-116.1 65.6-158.4C296 211.3 352.2 188 412 188s116.1 23.2 158.4 65.6S636 352.2 636 412s-23.3 116.1-65.6 158.4z"
                    />
                  </svg>
                </span>
              </span>
            </span>
          </div>
        </div>
        <div
          className="ant-space-item"
          style={
            Object {
              "marginRight": 8,
            }
          }
        >
          <div>
            <div
              className="ant-select ant-select-lg ant-select-borderless selectpin ant-select-single ant-select-show-arrow"
              onBlur={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
            >
              <div
                className="ant-select-selector"
                onClick={[Function]}
                onMouseDown={[Function]}
              >
                <span
                  className="ant-select-selection-search"
                >
                  <input
                    aria-activedescendant="rc_select_TEST_OR_SSR_list_0"
                    aria-autocomplete="list"
                    aria-controls="rc_select_TEST_OR_SSR_list"
                    aria-haspopup="listbox"
                    aria-owns="rc_select_TEST_OR_SSR_list"
                    autoComplete="off"
                    className="ant-select-selection-search-input"
                    id="rc_select_TEST_OR_SSR"
                    onChange={[Function]}
                    onCompositionEnd={[Function]}
                    onCompositionStart={[Function]}
                    onKeyDown={[Function]}
                    onMouseDown={[Function]}
                    onPaste={[Function]}
                    readOnly={true}
                    role="combobox"
                    style={
                      Object {
                        "opacity": 0,
                      }
                    }
                    type="search"
                    unselectable="on"
                    value=""
                  />
                </span>
                <span
                  className="ant-select-selection-item"
                  title="500070"
                >
                  500070
                </span>
              </div>
              <span
                aria-hidden={true}
                className="ant-select-arrow"
                onMouseDown={[Function]}
                style={
                  Object {
                    "WebkitUserSelect": "none",
                    "userSelect": "none",
                  }
                }
                unselectable="on"
              >
                <span
                  aria-label="environment"
                  className="anticon anticon-environment selectpin"
                  role="img"
                >
                  <svg
                    aria-hidden="true"
                    data-icon="environment"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    viewBox="64 64 896 896"
                    width="1em"
                  >
                    <path
                      d="M854.6 289.1a362.49 362.49 0 00-79.9-115.7 370.83 370.83 0 00-118.2-77.8C610.7 76.6 562.1 67 512 67c-50.1 0-98.7 9.6-144.5 28.5-44.3 18.3-84 44.5-118.2 77.8A363.6 363.6 0 00169.4 289c-19.5 45-29.4 92.8-29.4 142 0 70.6 16.9 140.9 50.1 208.7 26.7 54.5 64 107.6 111 158.1 80.3 86.2 164.5 138.9 188.4 153a43.9 43.9 0 0022.4 6.1c7.8 0 15.5-2 22.4-6.1 23.9-14.1 108.1-66.8 188.4-153 47-50.4 84.3-103.6 111-158.1C867.1 572 884 501.8 884 431.1c0-49.2-9.9-97-29.4-142zM512 880.2c-65.9-41.9-300-207.8-300-449.1 0-77.9 31.1-151.1 87.6-206.3C356.3 169.5 431.7 139 512 139s155.7 30.5 212.4 85.9C780.9 280 812 353.2 812 431.1c0 241.3-234.1 407.2-300 449.1zm0-617.2c-97.2 0-176 78.8-176 176s78.8 176 176 176 176-78.8 176-176-78.8-176-176-176zm79.2 255.2A111.6 111.6 0 01512 551c-29.9 0-58-11.7-79.2-32.8A111.6 111.6 0 01400 439c0-29.9 11.7-58 32.8-79.2C454 338.6 482.1 327 512 327c29.9 0 58 11.6 79.2 32.8C612.4 381 624 409.1 624 439c0 29.9-11.6 58-32.8 79.2z"
                    />
                  </svg>
                </span>
              </span>
            </div>
          </div>
        </div>
        <div
          className="ant-space-item"
          style={
            Object {
              "marginRight": 8,
            }
          }
        >
          <div
            className="login"
          >
            <button
              className="ant-btn ant-btn-primary ant-btn-lg ant-btn-background-ghost ant-btn-block login-button"
              onClick={[Function]}
              type="button"
            >
              <span>
                Login
              </span>
            </button>
          </div>
        </div>
        <div
          className="ant-space-item"
          style={Object {}}
        >
          <div
            className="globe"
          >
            <span
              aria-label="global"
              className="anticon anticon-global ant-dropdown-trigger globe globe"
              onMouseEnter={[Function]}
              onMouseLeave={[Function]}
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="global"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M854.4 800.9c.2-.3.5-.6.7-.9C920.6 722.1 960 621.7 960 512s-39.4-210.1-104.8-288c-.2-.3-.5-.5-.7-.8-1.1-1.3-2.1-2.5-3.2-3.7-.4-.5-.8-.9-1.2-1.4l-4.1-4.7-.1-.1c-1.5-1.7-3.1-3.4-4.6-5.1l-.1-.1c-3.2-3.4-6.4-6.8-9.7-10.1l-.1-.1-4.8-4.8-.3-.3c-1.5-1.5-3-2.9-4.5-4.3-.5-.5-1-1-1.6-1.5-1-1-2-1.9-3-2.8-.3-.3-.7-.6-1-1C736.4 109.2 629.5 64 512 64s-224.4 45.2-304.3 119.2c-.3.3-.7.6-1 1-1 .9-2 1.9-3 2.9-.5.5-1 1-1.6 1.5-1.5 1.4-3 2.9-4.5 4.3l-.3.3-4.8 4.8-.1.1c-3.3 3.3-6.5 6.7-9.7 10.1l-.1.1c-1.6 1.7-3.1 3.4-4.6 5.1l-.1.1c-1.4 1.5-2.8 3.1-4.1 4.7-.4.5-.8.9-1.2 1.4-1.1 1.2-2.1 2.5-3.2 3.7-.2.3-.5.5-.7.8C103.4 301.9 64 402.3 64 512s39.4 210.1 104.8 288c.2.3.5.6.7.9l3.1 3.7c.4.5.8.9 1.2 1.4l4.1 4.7c0 .1.1.1.1.2 1.5 1.7 3 3.4 4.6 5l.1.1c3.2 3.4 6.4 6.8 9.6 10.1l.1.1c1.6 1.6 3.1 3.2 4.7 4.7l.3.3c3.3 3.3 6.7 6.5 10.1 9.6 80.1 74 187 119.2 304.5 119.2s224.4-45.2 304.3-119.2a300 300 0 0010-9.6l.3-.3c1.6-1.6 3.2-3.1 4.7-4.7l.1-.1c3.3-3.3 6.5-6.7 9.6-10.1l.1-.1c1.5-1.7 3.1-3.3 4.6-5 0-.1.1-.1.1-.2 1.4-1.5 2.8-3.1 4.1-4.7.4-.5.8-.9 1.2-1.4a99 99 0 003.3-3.7zm4.1-142.6c-13.8 32.6-32 62.8-54.2 90.2a444.07 444.07 0 00-81.5-55.9c11.6-46.9 18.8-98.4 20.7-152.6H887c-3 40.9-12.6 80.6-28.5 118.3zM887 484H743.5c-1.9-54.2-9.1-105.7-20.7-152.6 29.3-15.6 56.6-34.4 81.5-55.9A373.86 373.86 0 01887 484zM658.3 165.5c39.7 16.8 75.8 40 107.6 69.2a394.72 394.72 0 01-59.4 41.8c-15.7-45-35.8-84.1-59.2-115.4 3.7 1.4 7.4 2.9 11 4.4zm-90.6 700.6c-9.2 7.2-18.4 12.7-27.7 16.4V697a389.1 389.1 0 01115.7 26.2c-8.3 24.6-17.9 47.3-29 67.8-17.4 32.4-37.8 58.3-59 75.1zm59-633.1c11 20.6 20.7 43.3 29 67.8A389.1 389.1 0 01540 327V141.6c9.2 3.7 18.5 9.1 27.7 16.4 21.2 16.7 41.6 42.6 59 75zM540 640.9V540h147.5c-1.6 44.2-7.1 87.1-16.3 127.8l-.3 1.2A445.02 445.02 0 00540 640.9zm0-156.9V383.1c45.8-2.8 89.8-12.5 130.9-28.1l.3 1.2c9.2 40.7 14.7 83.5 16.3 127.8H540zm-56 56v100.9c-45.8 2.8-89.8 12.5-130.9 28.1l-.3-1.2c-9.2-40.7-14.7-83.5-16.3-127.8H484zm-147.5-56c1.6-44.2 7.1-87.1 16.3-127.8l.3-1.2c41.1 15.6 85 25.3 130.9 28.1V484H336.5zM484 697v185.4c-9.2-3.7-18.5-9.1-27.7-16.4-21.2-16.7-41.7-42.7-59.1-75.1-11-20.6-20.7-43.3-29-67.8 37.2-14.6 75.9-23.3 115.8-26.1zm0-370a389.1 389.1 0 01-115.7-26.2c8.3-24.6 17.9-47.3 29-67.8 17.4-32.4 37.8-58.4 59.1-75.1 9.2-7.2 18.4-12.7 27.7-16.4V327zM365.7 165.5c3.7-1.5 7.3-3 11-4.4-23.4 31.3-43.5 70.4-59.2 115.4-21-12-40.9-26-59.4-41.8 31.8-29.2 67.9-52.4 107.6-69.2zM165.5 365.7c13.8-32.6 32-62.8 54.2-90.2 24.9 21.5 52.2 40.3 81.5 55.9-11.6 46.9-18.8 98.4-20.7 152.6H137c3-40.9 12.6-80.6 28.5-118.3zM137 540h143.5c1.9 54.2 9.1 105.7 20.7 152.6a444.07 444.07 0 00-81.5 55.9A373.86 373.86 0 01137 540zm228.7 318.5c-39.7-16.8-75.8-40-107.6-69.2 18.5-15.8 38.4-29.7 59.4-41.8 15.7 45 35.8 84.1 59.2 115.4-3.7-1.4-7.4-2.9-11-4.4zm292.6 0c-3.7 1.5-7.3 3-11 4.4 23.4-31.3 43.5-70.4 59.2-115.4 21 12 40.9 26 59.4 41.8a373.81 373.81 0 01-107.6 69.2z"
                />
              </svg>
            </span>
          </div>
        </div>
      </div>
    </div>
  </header>
  <main
    className="ant-layout-content content"
  >
    <div
      className="categories"
    >
      <div
        className="ant-row ant-row-left ant-row-middle"
        style={
          Object {
            "marginBottom": undefined,
            "marginLeft": undefined,
            "marginRight": undefined,
            "marginTop": undefined,
          }
        }
      >
        <div
          className="ant-col gutter-row  ant-col-xs-16 ant-col-sm-6 ant-col-md-5 ant-col-lg-3 ant-col-xl-3"
          style={Object {}}
        >
          <div
            className="ant-space ant-space-horizontal ant-space-align-center"
            style={Object {}}
          >
            <div
              className="ant-space-item"
              style={
                Object {
                  "marginRight": 8,
                }
              }
            >
              <img
                alt="fallback"
                src="./assets/images/clothing.png"
                style={
                  Object {
                    "height": "55px",
                    "width": "55px",
                  }
                }
              />
            </div>
            <div
              className="ant-space-item"
              style={Object {}}
            >
              <button
                className="ant-btn ant-btn-link ant-dropdown-trigger ant-dropdown-link category"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onMouseLeave={[Function]}
                type="button"
              >
                <span>
                  Clothing 
                </span>
                <span
                  aria-label="down"
                  className="anticon anticon-down"
                  role="img"
                >
                  <svg
                    aria-hidden="true"
                    data-icon="down"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    viewBox="64 64 896 896"
                    width="1em"
                  >
                    <path
                      d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
                    />
                  </svg>
                </span>
              </button>
            </div>
          </div>
        </div>
        <div
          className="ant-col gutter-row  ant-col-xs-16 ant-col-sm-6 ant-col-md-5 ant-col-lg-3 ant-col-xl-3"
          style={Object {}}
        >
          <div
            className="ant-space ant-space-horizontal ant-space-align-center"
            style={Object {}}
          >
            <div
              className="ant-space-item"
              style={
                Object {
                  "marginRight": 8,
                }
              }
            >
              <img
                alt="fallback"
                src="./assets/images/footwear.jpg"
                style={
                  Object {
                    "height": "55px",
                    "width": "55px",
                  }
                }
              />
            </div>
            <div
              className="ant-space-item"
              style={Object {}}
            >
              <button
                className="ant-btn ant-btn-link ant-dropdown-trigger ant-dropdown-link category"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onMouseLeave={[Function]}
                type="button"
              >
                <span>
                  Footwear 
                </span>
                <span
                  aria-label="down"
                  className="anticon anticon-down"
                  role="img"
                >
                  <svg
                    aria-hidden="true"
                    data-icon="down"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    viewBox="64 64 896 896"
                    width="1em"
                  >
                    <path
                      d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
                    />
                  </svg>
                </span>
              </button>
            </div>
          </div>
        </div>
        <div
          className="ant-col gutter-row  ant-col-xs-16 ant-col-sm-6 ant-col-md-5 ant-col-lg-3 ant-col-xl-3"
          style={Object {}}
        >
          <div
            className="ant-space ant-space-horizontal ant-space-align-center"
            style={Object {}}
          >
            <div
              className="ant-space-item"
              style={
                Object {
                  "marginRight": 8,
                }
              }
            >
              <img
                alt="fallback"
                src="./assets/images/bags.png"
                style={
                  Object {
                    "height": "55px",
                    "width": "55px",
                  }
                }
              />
            </div>
            <div
              className="ant-space-item"
              style={Object {}}
            >
              <button
                className="ant-btn ant-btn-link ant-dropdown-trigger ant-dropdown-link category"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onMouseLeave={[Function]}
                type="button"
              >
                <span>
                  Bags 
                </span>
                <span
                  aria-label="down"
                  className="anticon anticon-down"
                  role="img"
                >
                  <svg
                    aria-hidden="true"
                    data-icon="down"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    viewBox="64 64 896 896"
                    width="1em"
                  >
                    <path
                      d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
                    />
                  </svg>
                </span>
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="ant-carousel"
    >
      <div
        className="slick-slider slick-initialized"
        dir="ltr"
      >
        <div
          className="slick-list"
          onClick={[Function]}
          onKeyDown={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={null}
          onMouseMove={null}
          onMouseUp={[Function]}
          onTouchCancel={null}
          onTouchEnd={[Function]}
          onTouchMove={null}
          onTouchStart={[Function]}
          style={Object {}}
        >
          <div
            className="slick-track"
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onMouseOver={[Function]}
            style={
              Object {
                "WebkitTransform": "translate3d(0px, 0px, 0px)",
                "WebkitTransition": "",
                "msTransform": "translateX(0px)",
                "opacity": 1,
                "transform": "translate3d(0px, 0px, 0px)",
                "transition": "",
              }
            }
          >
            <div
              aria-hidden={true}
              className="slick-slide slick-cloned"
              data-index={-1}
              onClick={[Function]}
              style={
                Object {
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <div
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                >
                  <h3
                    className="carouseloffers"
                  >
                    Offer 4
                  </h3>
                </div>
              </div>
            </div>
            <div
              aria-hidden={false}
              className="slick-slide slick-active slick-current"
              data-index={0}
              onClick={[Function]}
              style={
                Object {
                  "outline": "none",
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <div
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                >
                  <h3
                    className="carouseloffers"
                  >
                    Offer 1
                  </h3>
                </div>
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide"
              data-index={1}
              onClick={[Function]}
              style={
                Object {
                  "outline": "none",
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <div
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                >
                  <h3
                    className="carouseloffers"
                  >
                    Offer 2
                  </h3>
                </div>
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide"
              data-index={2}
              onClick={[Function]}
              style={
                Object {
                  "outline": "none",
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <div
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                >
                  <h3
                    className="carouseloffers"
                  >
                    Offer 3
                  </h3>
                </div>
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide"
              data-index={3}
              onClick={[Function]}
              style={
                Object {
                  "outline": "none",
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <div
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                >
                  <h3
                    className="carouseloffers"
                  >
                    Offer 4
                  </h3>
                </div>
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide slick-cloned"
              data-index={4}
              onClick={[Function]}
              style={
                Object {
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <div
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                >
                  <h3
                    className="carouseloffers"
                  >
                    Offer 1
                  </h3>
                </div>
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide slick-cloned"
              data-index={5}
              onClick={[Function]}
              style={
                Object {
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <div
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                >
                  <h3
                    className="carouseloffers"
                  >
                    Offer 2
                  </h3>
                </div>
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide slick-cloned"
              data-index={6}
              onClick={[Function]}
              style={
                Object {
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <div
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                >
                  <h3
                    className="carouseloffers"
                  >
                    Offer 3
                  </h3>
                </div>
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide slick-cloned"
              data-index={7}
              onClick={[Function]}
              style={
                Object {
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <div
                  style={
                    Object {
                      "display": "inline-block",
                      "width": "100%",
                    }
                  }
                  tabIndex={-1}
                >
                  <h3
                    className="carouseloffers"
                  >
                    Offer 4
                  </h3>
                </div>
              </div>
            </div>
          </div>
        </div>
        <ul
          className="slick-dots slick-dots-bottom"
          onMouseEnter={null}
          onMouseLeave={null}
          onMouseOver={null}
          style={
            Object {
              "display": "block",
            }
          }
        >
          <li
            className="slick-active"
          >
            <button
              onClick={[Function]}
            >
              1
            </button>
          </li>
          <li
            className=""
          >
            <button
              onClick={[Function]}
            >
              2
            </button>
          </li>
          <li
            className=""
          >
            <button
              onClick={[Function]}
            >
              3
            </button>
          </li>
          <li
            className=""
          >
            <button
              onClick={[Function]}
            >
              4
            </button>
          </li>
        </ul>
      </div>
    </div>
    <div
      className="ant-divider ant-divider-horizontal ant-divider-with-text ant-divider-with-text-center"
      role="separator"
      style={
        Object {
          "fontSize": "x-large",
        }
      }
    >
      <span
        className="ant-divider-inner-text"
      >
        OFFERS
      </span>
    </div>
    <div
      className="offerscards"
    >
      <div
        className="ant-row ant-row-space-around ant-row-middle"
        style={
          Object {
            "marginBottom": undefined,
            "marginLeft": undefined,
            "marginRight": undefined,
            "marginTop": undefined,
          }
        }
      >
        <div
          className="ant-col ant-col-8"
          style={Object {}}
        >
          <div
            className="ant-card ant-card-bordered ant-card-hoverable"
            style={
              Object {
                "width": 400,
              }
            }
          >
            <div
              className="ant-card-cover"
            >
              <img
                alt="example"
                src="https://os.alipayobjects.com/rmsportal/QBnOOoLaAfKPirc.png"
              />
            </div>
            <div
              className="ant-card-body"
              style={Object {}}
            >
              <div
                className="ant-card-meta"
              >
                <div
                  className="ant-card-meta-detail"
                >
                  <div
                    className="ant-card-meta-title"
                  >
                    Bags 30% OFF
                  </div>
                  <div
                    className="ant-card-meta-description"
                  >
                    Shop Now
                  </div>
                </div>
              </div>
            </div>
          </div>
          ,
        </div>
        <div
          className="ant-col ant-col-8"
          style={Object {}}
        >
          <div
            className="ant-card ant-card-bordered ant-card-hoverable"
            style={
              Object {
                "width": 400,
              }
            }
          >
            <div
              className="ant-card-cover"
            >
              <img
                alt="example"
                src="https://os.alipayobjects.com/rmsportal/QBnOOoLaAfKPirc.png"
              />
            </div>
            <div
              className="ant-card-body"
              style={Object {}}
            >
              <div
                className="ant-card-meta"
              >
                <div
                  className="ant-card-meta-detail"
                >
                  <div
                    className="ant-card-meta-title"
                  >
                    Bags 30% OFF
                  </div>
                  <div
                    className="ant-card-meta-description"
                  >
                    Shop Now
                  </div>
                </div>
              </div>
            </div>
          </div>
          ,
        </div>
        <div
          className="ant-col ant-col-8"
          style={Object {}}
        >
          <div
            className="ant-card ant-card-bordered ant-card-hoverable"
            style={
              Object {
                "width": 400,
              }
            }
          >
            <div
              className="ant-card-cover"
            >
              <img
                alt="example"
                src="https://os.alipayobjects.com/rmsportal/QBnOOoLaAfKPirc.png"
              />
            </div>
            <div
              className="ant-card-body"
              style={Object {}}
            >
              <div
                className="ant-card-meta"
              >
                <div
                  className="ant-card-meta-detail"
                >
                  <div
                    className="ant-card-meta-title"
                  >
                    Bags 30% OFF
                  </div>
                  <div
                    className="ant-card-meta-description"
                  >
                    Shop Now
                  </div>
                </div>
              </div>
            </div>
          </div>
          ,
        </div>
      </div>
    </div>
  </main>
  <footer
    className="ant-layout-footer footer"
  >
    <div
      className="ant-space ant-space-horizontal ant-space-align-center"
      style={Object {}}
    >
      <div
        className="ant-space-item"
        style={
          Object {
            "marginRight": 8,
          }
        }
      >
        <button
          className="ant-btn ant-btn-link"
          onClick={[Function]}
          type="button"
        >
          <span>
            Terms & Conditions
          </span>
        </button>
      </div>
      <div
        className="ant-space-item"
        style={
          Object {
            "marginRight": 8,
          }
        }
      >
         |
      </div>
      <div
        className="ant-space-item"
        style={Object {}}
      >
        <button
          className="ant-btn ant-btn-link"
          onClick={[Function]}
          type="button"
        >
          <span>
            Made In India
          </span>
        </button>
      </div>
    </div>
  </footer>
</section>
`;
